packages/theme/package.json

```diff
@@ -1,17 +1,17 @@
 {
-    "name": "@spectrum-web-components/themes",
+    "name": "@spectrum-web-components/theme",
     "publishConfig": {
         "access": "public"
     },
     "repository": {
         "type": "git",
         "url": "https://github.com/adobe/spectrum-web-components.git",
-        "directory": "packages/themes"
+        "directory": "packages/theme"
     },
     "bugs": {
         "url": "https://github.com/adobe/spectrum-web-components/issues"
     },
-    "homepage": "https://adobe.github.io/spectrum-web-components/components/themes",
+    "homepage": "https://adobe.github.io/spectrum-web-components/components/theme",
     "keywords": [
         "spectrum css",
         "web components",
```

Comment:
Agree that letting this next version be 0.1.1 makes sense. To get there, set line 23 to "0.1.0", and then when we do `yarn lerna-publish` it'll manage the `fix` alteration outlined in the commits on its own. After that's there, I'm excited to approve/merge this!

Reply:
Done. Version modified!